### PR TITLE
SONAR-13925 Make SearchBox max length configurable

### PR DESCRIPTION
**Pull request by wouter-admiraal-sonarsource**
**Checklist**

* [x] Functional validation
* [ ] Documentation update
* [x] Update changelog

### CODE REVIEW

File: components/controls/SearchBox.tsx [hunk 1]
```diff
@@ -119,7 +120,7 @@ export default class SearchBox extends React.PureComponent<Props, State> {
   };
 
   render() {
-    const { loading, minLength } = this.props;
+    const { loading, minLength, maxLength = 100 } = this.props;
```
Comment (jeremy-davis-sonarsource) [inline components/controls/SearchBox.tsx]: NP: How about `DEFAULT_MAX_LENGTH` ?

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM

File: components/controls/SearchBox.tsx (hunk 1 above)
Comment (jeremy-davis-sonarsource) [reply components/controls/SearchBox.tsx]: nevermind :joy:

**Review by gregaubert — APPROVED** [review]
:+1: Looks great!

File: components/controls/SearchBox.tsx (hunk 1 above)
Comment (gregaubert) [reply components/controls/SearchBox.tsx]: Well, `maxLength` was already a constant no ? :smirk:

**Pull request merged**